demos/bootstrap.js

```diff
@@ -9,21 +10,61 @@ var script = scripts[ scripts.length - 1 ];
 var modules = script.getAttribute( "data-modules" );
 var pathParts = window.location.pathname.split( "/" );
 var effectsAll = [
-	"effect-blind",
-	"effect-bounce",
-	"effect-clip",
-	"effect-drop",
-	"effect-explode",
-	"effect-fade",
-	"effect-fold",
-	"effect-highlight",
-	"effect-puff",
-	"effect-pulsate",
-	"effect-scale",
-	"effect-shake",
-	"effect-size",
-	"effect-slide"
+	"effects/effect-blind",
+	"effects/effect-bounce",
+	"effects/effect-clip",
+	"effects/effect-drop",
+	"effects/effect-explode",
+	"effects/effect-fade",
+	"effects/effect-fold",
+	"effects/effect-highlight",
+	"effects/effect-puff",
+	"effects/effect-pulsate",
+	"effects/effect-scale",
+	"effects/effect-shake",
+	"effects/effect-size",
+	"effects/effect-slide"
 ];
+var widgets = [
+	"accordion",
+	"autocomplete",
+	"button",
+	"datepicker",
+	"dialog",
+	"draggable",
+	"droppable",
+	"menu",
+	"mouse",
+	"progressbar",
+	"resizable",
+	"selectable",
+	"selectmenu",
+	"slider",
+	"sortable",
+	"spinner",
+	"tabs",
+	"tooltip"
+];
+
+function getPath( module ) {
+	for ( var i = 0; i < widgets.length; i++ ) {
+		if ( widgets[ i ] === module ) {
+			return "widgets/" + module;
+		}
+	}
+	for ( var j = 0; j < effectsAll.length; j++ ) {
+		if ( effectsAll[ j ] === module ) {
+			return "effects/" + module;
```

Comment:
There is an error here it needs to check if it contains the string not `===` because this is processing when a demo requires a specific effect not all the effects.